### DISCUSSION TITLE
Linkable cards in `StatsOverviewWidget`

### DISCUSSION THREAD

**Discussion by jacksleight** [filamentphp/filament#1768] [Ideas]
Just been working on a stats widget using `StatsOverviewWidget` and would like to be able to link each stat card to the associated resource list page.

If you think this is a good idea I'm happy to work on a PR, but there are a few different ways it could be implemented:

1. A `url()` method on the `Card` component that links the entire card somewhere
2. A `descriptionUrl()` method on the `Card` component that just links the description text/icon somewhere
3. Allowing closures or `HtmlString` objects in `description()` and possibly `label()` and `value()` so links (or any custom HTML) could be added to those
4. Some combination of the above

Maybe 1 + 3 would offer the most flexibility?

What do you think?

**Comment by danharrin** [comment]
I agree! 1 + 3.

**Reply by ryangjchandler** [reply]
Let's string to `string | HtmlString` for the description/label/values too. Opt-in is best!

**Comment by cleargoal** [comment]
It's an interesting and good idea.
Did you make it?

**Comment by acharseth** [comment]
Agree! 1 + 2.
